golang/cosmos/x/swingset/config.go

```diff
@@ -0,0 +1,45 @@
+package swingset
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"path/filepath"
+)
+
+const (
+	ConfigPrefix = "swingset"
+	FlagSlogfile = "swingset.slogfile"
+)
+
+// DefaultConfigTemplate defines a default TOML configuration section for the SwingSet VM.
+// See https://github.com/cosmos/cosmos-sdk/issues/20097 for auto-synchronization ideas.
+const DefaultConfigTemplate = `
+
+	[swingset]
```

Comment:
Is the indentation here expected?

Reply:
probably not, but https://github.com/rollchains/tiablob/blob/main/tiasync/config.go has it too. I'll drop it.